appcelerator/titanium_mobile
# [TIMOB-12822] iOS: fixing timeout issue on simulator

Includes the new ios-sim version.

**TESTING INSTRUCTION**

Create a new project in TiStudio.
Build for simulator. 
Make sure the app launches correctly in the simulator and it does not timeout.
